docs/src/selectors.md

```diff
@@ -28,8 +28,8 @@ Selectors are strings that are used to create [Locator]s. Locators are used to p
   Learn more about [text selector][text].
 - CSS selector
   ```js
-  await page.locator('button').click();
-  await page.locator('#nav-bar .contact-us-item').click();
+  await page.locator("button").click();
```

Comment:
We actually prefer single quote `'` for stings in js/ts both in our code in the docs. Let's keep this page consistent with the rest of the docs.

Reply:
Understood, I will update this today (02/02). Thanks.